src/config.c

```diff
@@ -842,54 +879,14 @@ void configSetCommand(client *c) {
             return;
         }
     } config_set_special_field("client-output-buffer-limit") {
-        int vlen, j;
+        int vlen;
         sds *v = sdssplitlen(o->ptr,sdslen(o->ptr)," ",1,&vlen);
 
-        /* We need a multiple of 4: <class> <hard> <soft> <soft_seconds> */
-        if (vlen % 4) {
-            sdsfreesplitres(v,vlen);
+        if (updateClientOutputBufferLimit(&v[0], vlen, &errstr) == C_ERR) {
```

Comment:
why `&v[0]` and not just `v`?
```suggestion
        if (updateClientOutputBufferLimit(v, vlen, &errstr) == C_ERR) {
```